hydromt_wflow/workflows/river.py

```diff
@@ -451,7 +451,7 @@ def river_width(
         values = _precip(ds_like, flwdir=flwdir, logger=logger, **data)
     else:
         if predictor not in ds_like:
-            raise ValueError(f"required {predictor} variable missing in staticmaps.")
+            raise ValueError(f"required {predictor} variable missing in grid.")
```

Comment:
In these situations I find it often helpful to give some indication to which grid it referenced unless that will be clear from context. Up to you whether you want to do anything with this comment.